components/tabs/tab.js

```diff
@@ -0,0 +1,10 @@
+import { LitElement } from 'lit';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
```

Comment:
Per comment in the mixin, lets try to avoid using `RtlMixin` (since we want to get rid of it). If we need RTL specific styles, let's use the CSS logical properties instead.